code/style.css

```diff
@@ -1,4 +1,257 @@
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+.nav-item {
+  flex:1 0 10% ;
+  text-align: center;
+}
+.logo a, .nav-menu a {
+  display: block;
+  text-decoration: none;
+  letter-spacing: 0.5rem;
+  padding: 0.8rem;
+}
+
+.logo a {
+  color: black;
+}
+.big-news-link {
+  margin: 120px 10 px 20px 10px;
+  display: block;
+}
+.big-news {
+  display: flex;
+  border: 2px solid red;
+}
+.big-news-parent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.text{
+  text-align: center;
+  font-size: 20px;
+  background-color: white;
+  padding:5px;
+}
+.big-news-text{
+  text-align: center;
+  font-size: 20px;
+  background-color:none;
+  padding:5px;
+}
+.big-news img {
+  width: 350px;
+  height: 250px;
+  object-fit: cover;
+}
+.news-cards-parent{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+.news-card-child img {
+width: 300px;
+object-fit: cover;
+height: 200px;
+}
+.news-card-child {
+  padding-top: 40px;
+  background-color:white;
+  padding: 25px;
+  text-align: center;
+  box-shadow: 5px 5px 5px  whitesmoke;
+  width: 90%;
+  margin:5px 5px 15px 5px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+   justify-content: center;
+  align-items: center;
+}
+ .small-news {
+   display: flex;
+   flex-wrap: wrap;
+   justify-content: space-evenly;
+ }
+
+ .small-news a {
+   width: 30%
+ }
+ .news-item {
+  width: 100%;
+  height: 200px;
+  background: pink;
+  border: 2px solid hotpink;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.news-item img {
+  width: 70%;
+  height: 70%;
+  object-fit: cover;
+}
+
+.nav-menu a:hover{
+  background-color: blueviolet;
+  transform: scale(1.1);
+}
+.nav-manu a:active{
+  transform: scale(0.9);
+}
+label.logo {
+color: black;
+font-size: 100px;
+line-height: 100px;
+padding: 0 100px;
+font-weight: lighter;
+letter-spacing:  0.5rem;
+}
+nav ul {
+  float: right;
+  margin-right: 20px;
+}
+nav ul li {
+  display: inline-block;
+  line-height: 80px;
+  margin: 0 5px;
+}
+nav ul li a{
+  color: black;
+  font-size: 17px;
+  padding: 7px 13px;
+  border-radius: 3;
+  text-transform: uppercase;
+}
+a .active, a:hover{
+background: blueviolet;
+transition: .5s;
+} 
+/*ipad*/
+@media only screen and (min-width: 668px) and (max-width: 1024px) {
+ .news-cards-parent {
+  flex-direction: row;
+  justify-content: center;
+  flex-flow: row wrap;
+}
+.news-card-child{
+  width: 45%;
+  box-sizing: border-box;
+}
+.news-card-child img {
+  width: 80%;
+}
+}/*desktop*/
+@media only screen and (min-width: 1025px) {
+  .big-news-parent {
+    flex-direction: row;
+} 
+.big-news-parent img {
+  width: 50%;
+}
+.big-news-text{
+  text-align:center;
+}
+.news-cards-parent {
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-flow: row wrap;
+  width: 100%;
+}
+.news-card-child{
+  width: 20%;
+  box-sizing: border-box;
+}
+.text{
```

Comment:
This class could have a more specific name that describes where it belongs. Maybe "big-news-text-container"?